docs/visualization/visualize_2p_responses.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<p>This is not relevant to 2p</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>